python_modules/dagster/dagster/_scheduler/scheduler.py

```diff
@@ -298,22 +302,35 @@ def launch_scheduled_runs(
         elif location_entry.load_error:
             error_locations.add(location_entry.origin.location_name)
 
-    # Remove any schedule states that were previously created with DECLARED_IN_CODE
-    # and can no longer be found in the workspace (so that if they are later added
-    # back again, their timestamps will start at the correct place)
+    # Remove any schedule states that were previously created and can no longer
+    # be found in the workspace (so that if they are later added back again,
+    # their timestamps will start at the correct place)
     states_to_delete = [
         schedule_state
         for selector_id, schedule_state in all_schedule_states.items()
         if selector_id not in schedules
```

Comment:
note that the schedules dictionary here is only the *running* schedules - so as written this will also remove any stopped schedule rows 12 hours after you stop them, which I suspect is not what we want since it could then pop a schedule back to its DECLARED_IN_CODE value in code and turn the schedule back on?

The comment here is super misleading (not your fault) and the variable should probably be called "running_schedules", not "schedules"